api/RTCPeerConnection.json

```diff
@@ -210,6 +210,349 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "configuration_bundlePolicy_parameter": {
+          "__compat": {
+            "description": "<code>configuration.bundlePolicy</code> parameter",
+            "spec_url": "https://w3c.github.io/webrtc-pc/#dom-rtcconfiguration-bundlepolicy",
+            "support": {
+              "chrome": {
+                "version_added": "23"
+              },
+              "chrome_android": {
+                "version_added": "42"
```

Comment:
I don't know where I got this from in https://github.com/mdn/browser-compat-data/pull/12830#discussion_r759189379, I'm pretty sure it was from some research though.

Can you re-research it and update Chrome and Chrome for Android to match?

Reply:
It appears this was added in https://source.chromium.org/chromium/chromium/src/+/70aed24e4ae5f04ec31709b36353bfaed01547ad, which dates Chrome 42 based upon feature freeze dates (Feb 6, 2015 for Chrome 42)!

Reply:
Excellent! Can you update Chrome desktop data to match?